### DISCUSSION TITLE
How to place a letterbox?

### DISCUSSION THREAD

**Discussion by Maschga** [streetcomplete/StreetComplete#5779] [Q&A] [answered]
Hi,
I'd like to add a letterbox in the things-overlay.
Am I doing something wrong or is it not yet supported?

![Screenshot_20240802_121929_StreetComplete.png](https://github.com/user-attachments/assets/b3191ccf-2da6-42f1-8a50-2e87fa4ab34c)

**Comment by matkoniecz** [comment, marked as answer]
see https://github.com/streetcomplete/StreetComplete/issues/4916 (not done for technical reasons, once it will be resolved they will be listed)
